plsql/PlSqlParser.g4

```diff
@@ -21,7 +21,7 @@
 parser grammar PlSqlParser;
 
 options { tokenVocab=PlSqlLexer; }
-
+@members {boolean version12=true;}
```

Comment:
Is it possible to move runtime dependent code to super classes? See [JavaScript grammar](https://github.com/antlr/grammars-v4/tree/master/javascript) for example.

Reply:
I made the other changes.  I am not yet understanding how to make this change.  I read the example and it is not yet clear to me.  If I can figure it out, I will definitely make this change.

Reply:
I tried generating the javascript example and get errors trying to compile the java classes(I am using 4.7.1):

javac *.java
JavaScriptLexer.java:12: error: cannot find symbol
public class JavaScriptLexer extends JavaScriptBaseLexer {
                                     ^
  symbol: class JavaScriptBaseLexer
JavaScriptParser.java:12: error: cannot find symbol
public class JavaScriptParser extends JavaScriptBaseParser {
                                      ^
  symbol: class JavaScriptBaseParser
JavaScriptLexer.java:133: error: method does not override or implement a method from a supertype
	@Override

Reply:
It seems to me that you forgot to include the superclass files `JavaScriptBaseLexer` and `JavaScriptBaseParser` from java directory.

Reply:
Ok, I'll fix it by myself later.